client/ayon_ftrack/common/event_handlers/ftrack_action_handler.py

```diff
@@ -124,7 +124,11 @@ def _translate_event(
         ):
             _entities = [
                 item
-                for item in self._get_entities(event, session=session)
+                for item in self._get_entities(
+                    event,
+                    session=session,
+                    ignore=["socialfeed", "socialnotification", "team"]
```

Comment:
Not sure how big an average entity selection may be - but turning this into a `set` _may_ be beneficial for lookups?